network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -1,7 +1,27 @@
+
+from django.db import models
+
 from wagtail.core.models import Page
+from wagtail.core import blocks
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.contrib.table_block.blocks import TableBlock
 
 from ..mixin.foundation_metadata import FoundationMetadataPageMixin
 
+"""
+TODO: 
```

Comment:
Left over TODO. Could we possibly put this into a ticket for the current milestone and epic?

Reply:
Sure thing. I’m hoping to explain and resolve this at standup today!